core/src/main/java/org/apache/iceberg/MetadataUpdateParser.java

```diff
@@ -252,6 +254,10 @@ public static MetadataUpdate fromJson(JsonNode jsonNode) {
         return readAddSortOrder(jsonNode);
       case SET_DEFAULT_SORT_ORDER:
         return readSetDefaultSortOrder(jsonNode);
+      case SET_STATISTICS:
+        return readSetStatistics(jsonNode);
+      case REMOVE_STATISTICS:
+        return readRemoveStatistics(jsonNode);
```

Comment:
Looks like writing these is not yet supported?

I think this needs the write path and round-trip serialization tests.

Reply:
I missed that, added now. Thanks for pointing this out